tox.ini

```diff
@@ -24,12 +24,6 @@ deps =
 commands_pre = python -m spacy download en_core_web_sm
 commands = python -m pytest -m 'not spark and not complex' {posargs}
```

Comment:
while we're changing this, can we do like `{posargs:-m 'not spark and not complex'}` so that `tox -e py36 --` will run all tests?

Reply:
note to selves: `tox -e py36 -- ''` will run all tests